### PR TITLE
Update certificates for Active Directory

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

File: teradatalabs/centos6-java8-oracle-ldap/files/active-directory-certificate.crt [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-AAGjaTBnMBMGCSsGAQQBgjcUAgQGHgQAQwBBMA4GA1UdDwEB/wQEAwIBhjAPBgNV
-HRMBAf8EBTADAQH/MB0GA1UdDgQWBBRhUs5KPw8xotuxOQ34/9HlHAS8wjAQBgkr
-BgEEAYI3FQEEAwIBADANBgkqhkiG9w0BAQUFAAOCAQEACbU08ITkl4NSrIDmReDY
-U0zgnwNlcjyfDQ2eUfnrp3TpUDgGkDGkwAFtjlKngpaEKOWAUUUp/Akr7dKwpmef
-BEuHCqLyZdJB7vQYzcnA9kKgAafinyqSGQVQ3aMYWW/sRdtN2f2STyuGM0x0YeMx
-HuYrqCgoZLUXsyzn8Z1GRRUHSsyd5xdm5vbt5/kPBSV+9mHjS7zCyGJQNdiNWvoY
-jB6aCUs4FN5XgKSenCKg1B92zEJUgI70dKbcZyhzbJxjfnuXbuYNcpCc7ovFSzZg
-HulHmGFygMDnXzK2tzXW2iZ+ypZ8K9qdQeYZxyyiQYotbWPyD5vYMDqJyqv+s8Me
-LA==
+MIIGJTCCBQ2gAwIBAgITHQAAAAUM8gMo/ohRwwAAAAAABTANBgkqhkiG9w0BAQUF
```
Comment (ebd2) [inline teradatalabs/centos6-java8-oracle-ldap/files/active-directory-certificate.crt:2]: How long are the new certs good for, and do we have some guidelines for the next time we reissue to reduce the frequency of this problem?

**Review by ebd2 — COMMENTED** [review]
LGTM except for updating the README.md on the image with the cert so the next person who has to deal with this knows what's up.

File: teradatalabs/centos6-java8-oracle-ldap/files/active-directory-certificate.crt (hunk 1 above)
Comment (sanjay990) [reply teradatalabs/centos6-java8-oracle-ldap/files/active-directory-certificate.crt:2]: Its good for an year. We have the steps documented on internal WIKI.

**Comment by ebd2** [comment]
LGTM. Please squash the fixup onto the appropriate commit and you're good to merge the PR.

**Pull request merged**